### PR TITLE
Minor changes in People and AddPeople page

### PR DESCRIPTION
**Pull request by BAAHUBALI7781**

### CODE REVIEW

File: src/views/AddBlog.vue [hunk 1]
```diff
@@ -36,9 +36,7 @@
 </template>
 
 <script>
-import app from "firebase/app";
-import "firebase/firestore";
-import "firebase/auth";
+import firebase from "firebase";
```
Comment (shivam-51) [inline src/views/AddBlog.vue:39]: why? firebase has already been imported as "app". Look here...

File: src/views/AddBlog.vue [hunk 2]
```diff
@@ -52,10 +50,10 @@ export default {
   methods: {
     AddBlog() {
       // Add a new document in collection "blogs"
-      const db = app.firestore();
```
Comment (shivam-51) [inline src/views/AddBlog.vue:55]: ...here.
Wait, Why did you remove that? "Firebase" can be considered as a better representation but currently this change is not required.

File: src/views/AddPeople.vue [hunk 3]
```diff
@@ -20,12 +20,12 @@
             <br />
             <input
               class="input"
-              type="number"
+              type="phone"
```
Comment (shivam-51) [inline src/views/AddPeople.vue:23]: You know what's the difference between 'type="number"' and 'type= "phone"', right? When its number you cannot insert any alphabet in there, Its just a number.
Also you cannot just insert anything in type. type is a predefined attribute of html.

File: src/components/Homes/Intro.vue [hunk 4]
```diff
@@ -2,31 +2,26 @@
   <!-- Create Section -->
   <section id="create-section" class="py-5 mt-5">
     <div class="container">
-      <div class="mt-5">
```
Comment (shivam-51) [inline src/components/Homes/Intro.vue:5]: Why has ANY change been done in "Intro.vue"?

File: src/components/Family/First.vue [hunk 5]
```diff
@@ -87,6 +86,9 @@ export default {
   max-width: 30%;
   padding: 0 4px;
 }
+img{
+  height:230px;
+}
```
Comment (shivam-51) [inline src/components/Family/First.vue:91]: +1
Other than this, I do not think the changes done in the other files to be needed.

**Review by shivam-51 — CHANGES_REQUESTED** [review]

File: src/views/AddPeople.vue (hunk 3 above)
Comment (BAAHUBALI7781) [reply src/views/AddPeople.vue:23]: Yeah you are right! I wanted to use type="tel" instead used phone by mistake.
